packages/common/driver-definitions/src/protocol/consensus.ts

```diff
@@ -132,10 +132,19 @@ export interface IQuorumProposals {
 }
 
 /**
- * Interface combining tracking of clients as well as proposals in the Quorum.
  * @legacy
  * @alpha
  */
+export interface LegacyAlphaThatReferencesLegacyBetaAndPublic {
```

Comment:
move all existing legacy to beta, and then added a legacy alpha to illustrate. this will only be in packages/common/driver-definitions/api-report/driver-definitions.legacy.alpha.api.md